### ISSUE TITLE
plex player control is broken after api change

### ISSUE THREAD

**Issue by Hellowlol** [Hellowlol/HTPC-Manager#451]

**Comment by geolaw** [comment]
I don't want to post a duplicate but the title on this one is vague.
From the main page, I get a summary of new shows, new music, etc ... from my plex server 
clicking on a tv episode I would expect to go to that episode, or clicking on a recently added album in my music section ... but it all takes me to the plex movies page

Is that what this bug was filed for or should I open a new issue?

**Comment by Hellowlol** [comment]
Heh, the title was mostly a reminder for myself :P. What's broken is the player controls. Fx, say your watching plex on your phone on you lan, you could control the playback on the phone via htpc manager

**Comment by Hellowlol** [comment]
yes, make a new bug report. take a screenshot as im not sure i understood you issue. When you click a tv episode is it the plex movies tab inside htpc manager that is opened?

**Comment by ghost** [comment]
ok just updated to commit cb59c05ef3076cfe5d40034f44a6ee599c3f6551 and my plex player controls do not work either.

The controls show up, as per below:
![image](https://cloud.githubusercontent.com/assets/19606551/21986981/61f0876e-dc4d-11e6-83dc-7bdff7c1814b.png)

however none of the buttons do anything, logs entries below:

Log
2017-01-17 00:40:02	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:40:00	modules.plex	DEBUG	Sending skipNext to 192.168.2.191 value :
2017-01-17 00:39:59	modules.plex	DEBUG	Sending skipNext to 192.168.2.191 value :
2017-01-17 00:39:58	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:58	modules.plex	DEBUG	Sending skipPrevious to 192.168.2.191 value :
2017-01-17 00:39:57	modules.plex	DEBUG	Sending stepBack to 192.168.2.191 value :
2017-01-17 00:39:55	modules.plex	DEBUG	Sending stepBack to 192.168.2.191 value :
2017-01-17 00:39:54	modules.plex	DEBUG	Sending stop to 192.168.2.191 value :
2017-01-17 00:39:54	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:51	modules.plex	DEBUG	Sending pause to 192.168.2.191 value :
2017-01-17 00:39:51	modules.plex	DEBUG	Sending pause to 192.168.2.191 value :
2017-01-17 00:39:50	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:50	modules.plex	DEBUG	Sending pause to 192.168.2.191 value :
2017-01-17 00:39:49	modules.plex	DEBUG	Sending pause to 192.168.2.191 value :
2017-01-17 00:39:49	modules.plex	DEBUG	Sending pause to 192.168.2.191 value :
2017-01-17 00:39:49	modules.plex	DEBUG	Sending pause to 192.168.2.191 value :
2017-01-17 00:39:47	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:43	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:39	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:35	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:31	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:27	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:23	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:19	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:14	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:10	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:06	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:39:02	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:38:58	modules.plex	DEBUG	Fetching currently playing information
2017-01-17 00:38:54	modules.plex	DEBUG	Fetching currently playing information

**Comment by ghost** [comment]
Forgot to mention, I'm running Plex Version 1.3.3.3148
there is a docker update waiting for my plex container, however this issue seems to be from back in April, so I doubt updating to the very latest plex will fix.

**Comment by ghost** [comment]
I have no idea about player controls code, but it seems to not be sending any value each command? 
as it has 'Value:' with no actual value?
